client/packages/system/src/Name/Components/index.ts

```diff
@@ -4,3 +4,4 @@ export * from './CustomerSearchInput';
 export * from './CustomerSearchModal';
 export * from './InternalSupplierSearchInput';
 export * from './InternalSupplierSearchModal';
+export { getNameOptionRenderer } from './NameOptionRenderer';
```

Comment:
wasn't sure, does it matter if it's * or not ? for tree shaking ?

Reply:
tree shaking was working, but I prefer being targeted, like you've done